t/pt-stalk/pt-stalk.t

```diff
@@ -542,20 +542,68 @@ $retval = system("$trunk/bin/pt-stalk --no-stalk --run-time 2 --sleep 2 --dest $
 
 PerconaTest::wait_until(sub { !-f $pid_file });
 
-$output = `du -s $dest | wc -l`;
+$output = `ls -l $dest | wc -l`;
 
 $retval = system("$trunk/bin/pt-stalk --no-stalk --run-time 2 --dest $dest --retention-count 1 --pid $pid_file --iterations 1 -- --defaults-file=$cnf >$log_file 2>&1");
 
 PerconaTest::wait_until(sub { !-f $pid_file });
 
-$output = $output - `du -s $dest | wc -l`;
+$output = $output - `ls -l $dest | wc -l`;
 
 is(
    $output,
    0,
    "Retention test 5: retention by count works as expected"
 );
 
+# ###########################################################################
+# Test if option --system-only works correctly
```

Comment:
what if both options are given ? could you add test for that and check how your system behaves ?

Reply:
Test case added. See commit 3767e57e151fe7e3f9631d5c736334c1b8fc431b